pages/createUpdate.js

```diff
@@ -40,6 +41,8 @@ const CreateUpdateWrapper = styled(Flex)`
   }
 `;
 
+const UPDATE_TYPE = ['Normal Update', 'Changelog Entry'];
```

Comment:
This should be internationalized. I recommend defining this as an object:
```suggestion
const UPDATE_TYPE_MSGS = defineMessages({
  normal: { ... },
  changelog: { ... },
})
const UPDATE_TYPE = Object.keys(UPDATE_TYPE_MSGS);
```

Reply:
Done, although I had to use the `lodash.mapValues` method in addition to this as otherwise we are not taking the `defaultMessage` values from the internationalized messages.